deploy/10_1_gold_gbp_stablecoins.ts

```diff
@@ -0,0 +1,163 @@
+import type { BDStable } from "../typechain/BDStable";
+import type { HardhatRuntimeEnvironment } from "hardhat/types";
+import type { DeployFunction } from "hardhat-deploy/types";
+import * as constants from "../utils/Constants";
+import type { BdStablePool } from "../typechain/BdStablePool";
+import { bdStablesContractsDetails, formatAddress, getBdx, getDeployer, getTreasury } from "../utils/DeployedContractsHelpers";
+import { to_d12 } from "../utils/NumbersHelpers";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const initialDeployBDStables = Object.values(bdStablesContractsDetails).filter(stableDetails => ["bXAU", "bGBP"].includes(stableDetails.symbol));
+  for (const stableDetails of initialDeployBDStables) {
+    console.log(`Starting deployment: ${stableDetails.fiat} stable - ${stableDetails.symbol}`);
+
+    const deployer = await getDeployer(hre);
+    const treasury = await getTreasury(hre);
+
+    const bdx = await getBdx(hre);
+    const bdPoolLibraryDeployment = await hre.ethers.getContract("BdPoolLibrary");
+
+    await hre.deployments.deploy(stableDetails.symbol, {
+      from: deployer.address,
+      proxy: {
+        proxyContract: "OptimizedTransparentProxy",
+        execute: {
+          init: {
+            methodName: "initialize",
+            args: [
+              stableDetails.name,
+              stableDetails.symbol,
+              treasury.address,
+              bdx.address,
+              constants.initialBdstableMintingAmount(hre.network.name, stableDetails.symbol)
+            ]
+          }
+        }
+      },
+      contract: "BDStable",
+      args: []
+    });
+
+    const bdstable = (await hre.ethers.getContract(stableDetails.symbol)) as BDStable;
+    console.log(`${stableDetails.symbol} deployed to:`, bdstable.address);
+
+    await hre.deployments.deploy(stableDetails.pools.weth.name, {
+      from: deployer.address,
+      proxy: {
+        proxyContract: "OptimizedTransparentProxy",
+        execute: {
+          init: {
+            methodName: "initialize",
+            args: [
+              bdstable.address,
+              bdx.address,
+              formatAddress(hre, constants.wETH_address[hre.network.name]),
+              constants.wETH_precision[hre.network.name],
+              true
+            ]
+          }
+        }
+      },
+      contract: "BdStablePool",
```

Comment:
@tkeidar I believe by default `toggleRecollateralizePaused` is false.
So you need to call it on all 4 pools here in this deployment script.

Same goes for `buyBackPaused`